volume/local/local_unix.go

```diff
@@ -23,17 +24,19 @@ var (
 		"type":   true, // specify the filesystem type for mount, e.g. nfs
 		"o":      true, // generic mount options
 		"device": true, // device to mount from
+		"helper": true, // mount helper, either built-in (by default) or external
 	}
 )
 
 type optsConfig struct {
 	MountType   string
 	MountOpts   string
 	MountDevice string
+	MountHelper string
 }
 
 func (o *optsConfig) String() string {
-	return fmt.Sprintf("type='%s' device='%s' o='%s'", o.MountType, o.MountDevice, o.MountOpts)
+	return fmt.Sprintf("type='%s' device='%s' o='%s' helper='%s'", o.MountType, o.MountDevice, o.MountOpts, o.MountHelper)
```

Comment:
type='' / device='' / o=''  has same issues if values contain quote mark, do I need to fix them as well?